dev/features/hu.elte.txtuml.feature/feature.xml

```diff
@@ -116,6 +116,7 @@ This Agreement is governed by the laws of the State of New York and the intellec
       <import feature="org.eclipse.xtext.xbase.lib" version="2.8.4.v201508050135" match="perfect"/>
       <import feature="org.eclipse.xtext.ui" version="2.8.4.v201508050135" match="perfect"/>
       <import feature="org.eclipse.xtext.xtext.ui" version="2.8.4.v201508050135" match="perfect"/>
+      <import feature="org.eclipse.cdt" version="8.8.0.201509131935"/>
```

Comment:
@nagyandras95, have you considered setting a stricter version match rule? See cf962cd.